### PR TITLE
Pangram approach

### PR DESCRIPTION
**Pull request by bobahop**
Approach for Pangram

### CODE REVIEW

**Comment by bobahop** [comment]
@ErikSchierboom: C Pangram approach ready for review.

File: exercises/practice/pangram/.approaches/bitfield/content.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+A `char` is defined for each character to be tested in the loop.
+
+The `while` loop uses an assignment expression for its condition.
+The expression `(letter = *sentence)` has the value of `letter` after it has been been assigned the value of the dereferenced `sentence` pointer.
+If the value of `letter` is a null character (`'\0'`), then the while loop will not run.
+Note that, had the `sentence` pointer not been checked for being `NULL`, then trying to dereference it (`*sentence`) if it were a `NULL` pointer would be undefined behavior.
+It likely would result in a segmentation fault.
+
+
```
Comment (wolf99) [inline exercises/practice/pangram/.approaches/bitfield/content.md:51]: Remove doubled newline

File: exercises/practice/pangram/.approaches/bitfield/content.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+The [ASCII][ascii] value of the `letter` is compared with a range of ASCII values to check if it is one of the lowercase English letters.
+- The ASCII value for `a` is `97`, and for `z` is `122`.
```
Comment (wolf99) [inline exercises/practice/pangram/.approaches/bitfield/content.md:53]: Empty line needed before list

File: exercises/practice/pangram/.approaches/bitfield/content.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+- If the lowercase letter is subtracted by `97`, then `a` will result in `0`, because `97` minus `97` equals `0`.
+  `z` would result in `25`, because `122` minus `97` equals `25`.
+  So `a` would have `1` [shifted left][shift-left] 0 places (so not shifted at all) and `z` would have `1` shifted left 25 places.
+- The [bitwise OR operator][bitwise-or] is used to set the bit at that position in the `phrasemask`.
+
+If the `letter` is not a lowercase English letter,
+the ASCII value of the `letter` is compared with a range of ASCII values to check if it is one of the uppercase English letters.
+- The ASCII value for `A` is `65`, and for `Z` is `90`.
```
Comment (wolf99) [inline exercises/practice/pangram/.approaches/bitfield/content.md:61]: Need empty line before list here

**Review by wolf99 — DISMISSED** [review]
Really good Bob!
Just a few small comments re markdown linting

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**